Makefile

```
@@ -24,6 +24,10 @@ export TF_IN_AUTOMATION="true"
 export TF_VAR_MATURITY=${MATURITY}
 export TF_VAR_DEPLOY_NAME=${DEPLOY_NAME}
 PYTHON_VER ?= python3
+
+CIRRUS_CORE_VERSION = $(or $(shell git tag --points-at HEAD | head -n1),$(shell git rev-parse --short HEAD))
+CIRRUS_DAAC_VERSION = $(or $(shell git -C $(DAAC_DIR) tag --points-at HEAD | head -n1),$(shell git -C $(DAAC_DIR) rev-parse --short HEAD))
```

I think you'll want to use the `:=` operator here to evaluate these commands only once instead of evaluating them every time they're referenced.

```suggestion
CIRRUS_CORE_VERSION := $(or $(shell git tag --points-at HEAD | head -n1),$(shell git rev-parse --short HEAD))
CIRRUS_DAAC_VERSION := $(or $(shell git -C $(DAAC_DIR) tag --points-at HEAD | head -n1),$(shell git -C $(DAAC_DIR) rev-parse --short HEAD))
```